script.js

```diff
@@ -0,0 +1,30 @@
+$(document).ready(function() {
+
+	$('li').on('click', function(){
+		console.log("You clicked.");
+	});
+	$('#submitnew').on('click', function(e) {
+		e.preventDefault();
+		var $newThing = $('#newtodo input').val();
+		console.log ($newThing);
+		$('#item').val("");
+	
+
```

Comment:
There's an issue here that's causing your clear function to execute only after the submitnew function has executed. We simply need to move the closing brackets for the submitnew function up. Otherwise, you can't clear the first three items when initially loading the page.
```
$('#submitnew').on('click', function(e) {
 		e.preventDefault();
 		var $newThing = $('#newtodo input').val();
 		console.log ($newThing);
 		$('#item').val("");
                $('ul').append('<li><span>X </span>' + $newThing + '</li>');
});

$('ul').on('click', 'li span', function () {
         console.log("Item clicked");
         $('#item').val("");
         $(this).parent().remove();
 	});
  
$(function() {
     $('ul').sortable();
     $('ul').disableSelection();
});
```